sway/tree/workspace.c

```diff
@@ -572,14 +572,18 @@ bool workspace_switch(struct sway_workspace *workspace,
 		active_ws = focus->sway_container->pending.workspace;
 	}
 
-	if (!no_auto_back_and_forth && config->auto_back_and_forth && active_ws
-			&& active_ws == workspace && seat->prev_workspace_name) {
+	if (config->auto_back_and_forth && active_ws &&
+			active_ws == workspace && seat->prev_workspace_name) {
 		struct sway_workspace *new_ws =
-			workspace_by_name(seat->prev_workspace_name);
-		workspace = new_ws ?
-			new_ws :
-			workspace_create(NULL, seat->prev_workspace_name);
+				workspace_by_name(seat->prev_workspace_name);
+		workspace = new_ws ? new_ws
+						   : workspace_create(NULL, seat->prev_workspace_name);
```

Comment:
Style: don't align, instead use one indentation level for continuation lines.

Reply:
Fixed. But `clang-format` insists that the aligned way of formatting is the right one, so maybe there is a setting to change that. (Running it on the entire file makes lots of other diffs as well.)